VetClinic.WebApi/ViewModels/PetViewModels/PetViewModel.cs

```diff
@@ -5,13 +5,14 @@ namespace VetClinic.WebApi.ViewModels
 {
     public class PetViewModel
     {
-        public int Id { get; set; }
+        public int? Id { get; set; }
```

Comment:
In Post method don't need Id
[HttpPost]
        public async Task<IActionResult> InsertPet(PetViewModel petViewModel)